### PR TITLE
[99] Show Issue Title

### PR DESCRIPTION
**Pull request by AZielinsky95**
### Summary 

If the commit we receive references an issue number in the format `[99]`, replace the issue number with the Issues title. 

This works for as many matches of [{issue number}] there are in a commit message.

Raw Commit Message:
`[1] another one`

Coda Table Entry:
Screen Shot 2022-03-30 at 9 22 49 AM

Github Issue:
Screen Shot 2022-03-30 at 9 28 08 AM

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -82,4 +101,14 @@ async function run(): Promise<void> {
   }
 }
 
+async function replaceAsync(str, regex, asyncFn) {
+  const promises = [] as any
+  str.replace(regex, (match, ...args) => {
+    const promise = asyncFn(args[0])
```
Comment (jenncoop) [inline src/main.ts:107]: Is there a reason that match is omitted and we're only passing the first arg to asyncFn?

**Review by jenncoop — APPROVED** [review]
Looks reasonable! See my comments/questions 🚀

File: src/main.ts (hunk 1 above)
Comment (AZielinsky95) [reply src/main.ts:107]: The match includes the brackets where as the first arg references the capture group of the regex which is just the issue number.

**Comment by AZielinsky95** [comment]
Done! Both [99] [#99] will work now

**Pull request merged**